src/game/construct_types.h

```diff
@@ -16,5 +16,10 @@ struct ConstructData {
 namespace Constructs {
     const static ConstructData CHEST = { ConstructType::CHEST, 50, "Chest", "A treasure chest full of gold.", "test.png" };
     const static ConstructData SPIKES = { ConstructType::SPIKES, 50, "Spikes", "It was me, spikes.", "dio.jpg" };
-    const static ConstructData REMOVE = { ConstructType::REMOVE, 0, "Remove", "Remove a block.", "Tomato.jpg" };
+    const static ConstructData REMOVE = { ConstructType::REMOVE, 200, "Remove", "Remove a block.", "Tomato.jpg" };
+    const static std::map<ConstructType, ConstructData> CONSTRUCTS = {
```

Comment:
I'm mostly unsettled about the redundancy in having `CHEST -> { ConstructType::CHEST ...}` and `ConstructType::CHEST -> CHEST`. Can we remove all the static ConstructData declarations and instead just use `CONSTRUCTS[ConstructType::CHEST] -> {50, "Chest"}` every time instead?

Don't worry too much about it if it's too much work, only `|constructs|` lines of bloat anyways.